tests/ltp/ext2fs_fs_tests_passed.txt

```diff
@@ -35,7 +35,6 @@
 /ltp/testcases/kernel/syscalls/fchown/fchown03
 /ltp/testcases/kernel/syscalls/fchown/fchown05
 /ltp/testcases/kernel/syscalls/fchownat/fchownat01
-/ltp/testcases/kernel/syscalls/fchownat/fchownat02
```

Comment:
This test is passing for me locally. Could you post what error you're seeing on your machine?

Reply:
```
tests/ltp$ sudo make one FS=ramfs TEST=/ltp/testcases/kernel/syscalls/fchownat/fchownat02
rm -rf ramfs.appdir
mkdir -p ramfs.appdir//ltp/testcases/kernel/syscalls/fchownat
cp appdir//ltp/testcases/kernel/syscalls/fchownat/fchownat02 ramfs.appdir//ltp/testcases/kernel/syscalls/fchownat/fchownat02
cp -r appdir/etc  ramfs.appdir/
/home/feliu/workspace2/mystikos/build/bin/myst mkcpio ramfs.appdir ramfs
sudo -E   /home/feliu/workspace2/mystikos/build/bin/myst exec-sgx  ramfs /ltp/testcases/kernel/syscalls/fchownat/fchownat02
fchownat02    1  TFAIL  :  fchownat02.c:111: fchownat() failed, errno=22 : Invalid argument: TEST_ERRNO=EINVAL(22): Invalid argument
Makefile:197: recipe for target 'one' failed
make: *** [one] Error 1
```

Reply:
This might be related to uid 1000/user1000 not being present.

Reply:
> 
> 
> This might be related to uid 1000/user1000 not being present.

@vtikoo Right, `user1000` wasn't created on my host. I have created #491 to address this issue.